lib/game.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'player'
+require_relative 'board'
+
+class Game
+
+  attr_reader :player_1, :player_2, :board, :current_player
+
+  def initialize
+    @selected_token = nil
+    @player_1 = welcome_player("player_1")
+    @player_2 = welcome_player("player_2")
+    @board = Board.new()
```

Comment:
Please just pass an instantiated Board as a parameter to the initialize method. Like the Player class, you shouldn't be instantiating a class inside another class.